internal/hns/hnsendpoint.go

```diff
@@ -2,9 +2,9 @@ package hns
 
 import (
 	"encoding/json"
-	"net"
-
 	"github.com/sirupsen/logrus"
+	"net"
```

Comment:
These should be with platform imports

Reply:
gofmt -s didn't like that, sorry, I've fixed